R/tune_bayes.R

```diff
@@ -618,12 +618,7 @@ more_results <- function(object, resamples, candidates, metrics, control, param_
         param_info = param_info,
         grid = candidates,
         metrics = metrics,
-        control = control_grid(
-          verbose = FALSE,
-          extract = control$extract,
-          save_pred = control$save_pred,
-          event_level = control$event_level
-        )
+        control = parsnip::condense_control(control, control_grid())
```

Comment:
This will pass more argument through then previously, but it seems like the right thing to do

Reply:
I realized that we don't even need this as `tune_grid()` does this under the hood